tools/RAiDER/models/hres.py

```diff
@@ -0,0 +1,235 @@
+import datetime
+
+import numpy as np
+from pyproj import CRS
+
+from RAiDER import utilFcns as util
+from RAiDER.models.weatherModel import WeatherModel
+
+
+class HRES(WeatherModel):
+    '''
+    Implement ECMWF models
+    '''
+
+    def __init__(self):
+        # initialize a weather model
+        WeatherModel.__init__(self)
+
+        # model constants
+        self._k1 = 0.776   # [K/Pa]
+        self._k2 = 0.233   # [K/Pa]
+        self._k3 = 3.75e3  # [K^2/Pa]
+
+        # 9 km horizontal grid spacing
+        self._lon_res = 9. / 111
+        self._lat_res = 9. / 111
+        self._x_res = 9. / 111
+        self._y_res = 9. / 111
```

Comment:
Can you add a comment that this is only used for extending the download-buffer, i.e. not in subsequent processing.